root/etc/nginx/nginx.conf.tpl

```diff
@@ -1,18 +1,35 @@
 daemon off;
 
+
 http {
     include /etc/nginx/proxy.conf;
     include /etc/nginx/mime.types;
 
     server {
-	    client_body_temp_path /tmp/nginx_client_temp 1 2;
-        listen 8080 default_server;
+        client_body_temp_path /tmp/nginx_client_temp 1 2;
+        listen 8080 default_server ssl;
         resolver ${DNS_SERVER};
         access_log /dev/stdout;
 
+	ssl on;
+	server_name "";
+        ssl_certificate /etc/nginx/server.crt;
```

Comment:
This would require change in openshift template. Currently we termnate https on edge.